tests/foreman/api/test_oscap_tailoringfiles.py

```diff
@@ -0,0 +1,69 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Tailoring Files
+
+:Requirement: tailoringfiles
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: SCAPPlugin
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.config import settings
+from robottelo.datafactory import gen_string
+from robottelo.decorators import tier1
+from robottelo.helpers import file_downloader
+from robottelo.test import APITestCase
+
+
+class TailoringFileTestCase(APITestCase):
+    """Implements Tailoring Files tests in API."""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TailoringFileTestCase, cls).setUpClass()
+        cls.oscap_tailoring_path = file_downloader(file_url=settings.oscap.tailoring_path)[0]
+        cls.org = entities.Organization().create()
+        cls.loc = entities.Location(organization=[cls.org]).create()
+
+    @tier1
+    def test_positive_crud_tailoringfile(self):
+        """Perform end to end testing for oscap tailoring files component
+
+        :id: 2441988f-2054-49f7-885e-3675336f712f
+
+        :expectedresults: All expected CRUD actions finished successfully
+
+        :CaseImportance: Critical
+        """
+        name = gen_string('alpha')
+        new_name = gen_string('alpha')
+        original_filename = gen_string('alpha')
+        scap = entities.TailoringFile(
+            name=name,
+            scap_file=self.oscap_tailoring_path,
+            organization=[self.org],
+            location=[self.loc],
+        ).create()
+        assert entities.TailoringFile().search(query={'search': 'name={}'.format(name)})
```

Comment:
You could use the `f-string`(An Improved String Formatting Syntax) instead of `.format`:
like: 
`assert entities.TailoringFile().search(query={'search': f'name={name}'})`

It is a great new way to format strings. Not only are they more readable, more concise, and less prone to error than other ways of formatting, but they are also faster.

Reply:
@hkx303 yes, it looks much better now.